css/style.css

```diff
@@ -1,19 +1,223 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+  --dark-orange: #f15b2a;
+  --dark-grey: #676767;
+}
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  background: #eee;
+}
+
+/* ---------- Header style ---------- */
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 5vw;
+  align-items: center;
+}
+
+header img {
+  width: 30px;
+}
+
+nav {
+  width: 60%;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+  margin-left: 0;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 10px;
+}
+
+li a {
+  text-decoration: none;
+  color: lightslategray;
+  font-weight: 400;
+  cursor: pointer;
+}
+
+li:nth-child(1) a {
+  color: black;
+}
+
+li a:hover {
+  color: var(--dark-orange);
+}
+
+/* ---------- Main style (SECTION 1) ---------- */
+
+.introducing-section {
+  color: white;
+  background-image: url("../img/first-background.jpg");
+  background-size: cover; /* Cover the entire area */
+  background-position: center; /* Center the image */
+  background-repeat: no-repeat; /* Do not repeat the image */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 95vh;
+}
+
+.introducing-section h1 {
+  font-size: 5rem;
+  font-weight: 300;
+}
+
+.introducing-section p {
+  color: wheat;
+  margin-bottom: 2.2em;
+  font-size: 2.5rem;
+  font-weight: 100;
+}
+
+.introducing-section a {
+  padding: 1em;
+  background-color: var(--dark-orange);
+  color: white;
+  text-decoration: none;
+  border-radius: 3px;
+  font-size: 1.5rem;
+}
+
+/* ---------- Main style (SECTION 2) ---------- */
+
+.options-section {
+  display: flex;
+  flex-direction: column;
+  padding-top: 8em;
+  padding-left: 6em;
+  padding-right: 6em;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.options-section h2 {
+  font-size: 3rem;
+  font-weight: 300;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.options-icons {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 5em;
+  margin-bottom: 5em;
```

Comment:
I think I would move this one to the section and use a padding instead. Do you want it to be the same as the margin between the heading and the option items, or the same as the padding to the top of the section?

Reply:
again, I did it because of media queries